### DISCUSSION TITLE
Went from a very long README to this!

### DISCUSSION THREAD

**Discussion by wilhelm-murdoch** [retypeapp/retype#189] [Show and tell]
Saying I love Retype would be an absolute understatement. What a great project! It took me a couple days, but I've fully-migrated from my standard `README.md` file to my new site for my little side project; [Chamber](https://github.com/wilhelm-murdoch/chamber).

Check it out [here](https://chamber.wilhelm.codes)!

Thank you so much for all the hard work you've put into this project. Keep it up! 😃

**Comment by geoffreymcgill** [comment]
Retype has the best community. :raised_hands: 🤗 

Your project and new website looks awesome. Super usable, clean, and clear. Nice work!

> It took me a couple days

Did it take a couple days because you wanted to break-up one single `README.md` into multiple pages and that just took time to plan/organize/implement? Any suggestions on what we could do to make that process more efficient for you? 

I can certainly see reorganizing a large `README.md` would require some time, although our goal is to make that upgrade process as quick and as integrated as we can.

Any ideas you have, even just the smallest enhancements would be of interest. We are truly interested in every bit of optimization.

Follow-up question: What in particular did you like about Retype? Those details would be interesting to know.

Thanks for using Retype!

**Comment by wilhelm-murdoch** [comment]
> Did it take a couple days because you wanted to break-up one single README.md into multiple pages and that just took time to plan/organize/implement? Any suggestions on what we could do to make that process more efficient for you?

To be fair, it took a few days simply because I was just moving a bit slower than I usually do with these things. I was playing with different approaches and toying with all the features. I also added quite a bit of extra documentation as I went.

> Any ideas you have, even just the smallest enhancements would be of interest. We are truly interested in every bit of optimization.

1. A bit more flexibility in modifying the CSS? I know this isn't as straight-forward as some would think.
2. I wouldn't mind being able to collapse the "contents" column to widen the content of the page a bit. Or, if there aren't any items to populate that column, just auto-collapse it?
3. Widescreen support has already been brought up, but I understand it's not as easy as it sounds considering all the design elements that would need rebalancing.

Honestly, nothing that really hasn't already been brought up. This has been a great experience so far!

> What in particular did you like about Retype?

1. The default style, both dark and light modes, are just 😚 👌. 
2. Local development mode is extremely responsive.
3. Really digging the code snippet functionality. Being able to specify "regions" over "ranges" is a massive QoL feature.
4. Documentation is refreshingly-easy to follow.

This has been great so far! No real complaints. Keep up the great work!
